composer.json

```diff
@@ -18,7 +18,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "~2.3",
+        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7",
```

Comment:
we drop compatibility with 2.4 and 2.5 as they are not fixed. 
i also updated the .travis.yml to not attempt to build those versions.

btw, `,` means AND, `||` means OR.